modules/cnf-monitoring-fast-events-metrics-using-cli.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-monitoring-fast-events-metrics-using-cli_{context}"]
+= Monitoring PTP fast event notification metrics using the CLI
+
+You can monitor fast events bus metrics directly from pods in the `cloud-native-events` namespace. To view the metrics for a node, perform the following procedure.
+
+[NOTE]
+====
+PTP fast event notification metrics are also available from the `linuxptp` container and Prometheus. The PTP Operator is configured to send metrics to Prometheus by default.
+====
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install and configure the PTP Operator.
+
+.Procedure
+
+. Get the list of active `cloud-native-events` pods.
++
+[source,terminal]
+----
+$ oc get pods -n cloud-native-events
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                     READY   STATUS    RESTARTS   AGE
+cloud-native-consumer-deployment-4s5zc   3/3     Running   0          8h
+cloud-native-consumer-deployment-gcq7c   3/3     Running   0          8h
+----
+
+. Use `oc exec` to access the metrics for the required `cloud-native-event-consumer` pod, for example:
++
+[source,terminal]
+----
+$ oc exec -n cloud-native-events -it cloud-native-consumer-deployment-4s5zc -c cloud-native-event-consumer -- curl 127.0.0.1:9091/metrics
```

Comment:
```
oc logs -f linuxptp-daemon-qhfg7 -n openshift-ptp -c cloud-event-proxy  -- curl 127.0.0.1:9091/metrics

```

Reply:
@aneeshkp Are you sure this `oc logs` command is correct? don't we need to `oc exec` to run curl against the cloud-event-proxy container? ie., 
`oc exec -f linuxptp-daemon-qhfg7 -n openshift-ptp -c cloud-event-proxy  -- curl 127.0.0.1:9091/metrics`